- 运行 `rustup update` 时，遇到了 `error: failed to set permissions for '/Users/user/.cargo/bin/rustup-init': No such file or directory (os error 2)` 报错
	- 尝试[运行 `rustup self update` 解决了问题](https://github.com/rust-lang/rustup/issues/671#issuecomment-311552541)